repl/scala-2.10/src/main/scala/org/apache/spark/repl/SparkILoop.scala

```diff
@@ -1071,8 +1071,16 @@ object SparkILoop extends Logging {
     if (envJars.isDefined) {
```

Comment:
If you want to do something else, you could file a separate bug to remove this. Not only this is deprecated but it doesn't even exist in the 2.11 version, so might as well get rid of it.